index.js

```diff
@@ -1,18 +1,44 @@
-/***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+/***** Deliverable 1 *****/ //[DONE]
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
-/***** Deliverable 2 *****/
+/***** Deliverable 2 *****/ //[DONE]
 
+header.style.color = "blue"
 
+/***** Deliverable 3 *****/ //[DONE]
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.getElementById("profile")
 
-/***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+const image = profile.getElementsByTagName("img")[0]
+image.src = traveler.photo
+image.alt = traveler.name
 
+const h2 = profile.getElementsByTagName("h2")[0]
+h2.innerHTML = traveler.name
 
-/***** Deliverable 4 *****/
+const em = profile.getElementsByTagName("em")[0]
+em.innerHTML = traveler.nickname
 
+/***** Deliverable 4 *****/ //[DONE]
 
+const animal_sightings = traveler.animalSightings
```

Comment:
camelCase is the convention for JavaScript variables and function names (:

```suggestion
const animalSightings = traveler.animalSightings
```